korgw/src/mingwX64Main/kotlin/com/soywiz/korgw/DefaultGameWindowMingwX64.kt

```diff
@@ -440,6 +445,11 @@ fun WndProc(hWnd: HWND?, message: UINT, wParam: WPARAM, lParam: LPARAM): LRESULT
             val y = (lParam.toInt() ushr 16) and 0xFFFF
             mouseMove(x, y, wParam.toInt())
         }
+        _WM_MOUSEWHEEL -> {
+            val type = com.soywiz.korev.MouseEvent.Type.SCROLL
+            val scrollDeltaY = ((wParam.toInt() ushr 16) and 0xFFFF).toByte().toDouble()
+            windowsGameWindow.mouseEvent(type, mouseX, mouseY, 8, wParam.toInt(), scrollDeltaY)
```

Comment:
So the values of scrollDeltaY are between -128 and +128? Should be they -1.0, +1.0?

Reply:
Maybe `.convertRange(-128.0, +127.0, -1.0, +1.0)` ?

Reply:
The value is either -128, or 128. I don't know if it's the same on different mingwX64 targets (like windows 7, windows 10, etc.). I think 128 is a number of pixels to scroll

Reply:
https://docs.microsoft.com/en-us/windows/win32/inputdev/wm-mousewheel

> The high-order word indicates the distance the wheel is rotated, expressed in multiples or divisions of WHEEL_DELTA, which is 120. A positive value indicates that the wheel was rotated forward, away from the user; a negative value indicates that the wheel was rotated backward, toward the user.
> The low-order word indicates whether various virtual keys are down. This parameter can be one or more of the following values.

Reply:
Answered in Slack:
"Well, we don't know what values are provided on macos and linux native. So I don't think we should convert values right now. I think we should implement scrolling on all the targets and then check these scroll values and adjust all of them to the same value"

Reply:
It's already implemented on the other targets I guess. At least on linux